_health-care/_js/components/form-elements/FixedTable.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+class FixedTable extends React.Component {
+  render() {
+    const rowElements = this.props.rows.map((obj, index) => {
+      return (
+        <div className="row" key={index}>
+          <div className="small-9 columns">
```

Comment:
Let's remove these Foundation classes, they're not necessary here. We can entirely remove line 8 and then just remove the `className="row"` in line 7.